packages/components/Avatar/src/index.ts

```diff
@@ -1,3 +1,4 @@
 export * from './Avatar';
 export * from './Avatar.types';
 export { useAvatar } from './useAvatar';
+export { getHashCodeWeb } from './getHashCode';
```

Comment:
Does this mean we always export `getHashCodeWeb` to all customers? Is that something we want to do?